modules/virt-about-resource-quota-limits.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-specifying-nodes-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-setting-resource-quota-limits_{context}"]
```

Comment:
Please rename the module file itself to match the new title/ID (and of course, update it in the assembly when you do). Thanks!

Reply:
Done.